dnscrypt-proxy/main.go

```diff
@@ -101,6 +119,14 @@ func (app *App) Start(service service.Service) error {
 }
 
 func (app *App) AppMain() {
+	if err := ConfigLoad(app.proxy, app.flags); err != nil {
+		dlog.Fatal(err)
+	}
+	if err := app.proxy.InitPluginsGlobals(); err != nil {
+		dlog.Fatal(err)
+	}
+	app.quit = make(chan struct{})
+	app.wg.Add(1)
```

Comment:
This is the only strange piece left in this PR. We have a waitgroup and quit channel sitting here that aren't really necessary. But i presume these will be used at some point in a future PR to manage a clean shutdown.

Reply:
I don't even remember what the `quit` channel is for :) Probably not necessary indeed.

The waitgroup is indeed for a clean shutdown. It's not really used here, but will with @vbauerster changes.